build.sc

```diff
@@ -548,6 +538,15 @@ object scalanativelib extends MillModule {
   }
 }
 
+object bsp extends MillModule {
+  override def compileModuleDeps = Seq(scalalib, scalajslib, scalanativelib)
+  def ivyDeps = Agg(
+    Deps.bsp,
+    Deps.ujsonCirce,
```

Comment:
How difficult would it be to remove this Circe dependency? I'd like to avoid pulling in a heavyweight dependency tree if there's an easy way around it

Reply:
I removed it and it compiled so I guess it was not used.